src/themes/pre-bootstrap-customization.scss

```diff
@@ -1,48 +1,5 @@
-@import './common-variables.scss';
+// Customize Variables that get imported before the _variables file. Thus the _variables file
+// can depend on values here.
+$brand-primary: #c71b0a;
 
-
-//== Scaffolding
-//
-//## Settings for some of the most global styles.
-
-//** Background color for `<body>`.
-$body-bg:               #fff !default;
-//** Global text color on `<body>`.
-$text-color:            $brand-primary !default;
-
-//** Global textual link color.
-$link-color:            $brand-success !default;
-//** Link hover color set via `darken()` function.
-$link-hover-color:      $link-color !default;
-//** Link hover decoration.
-$link-hover-decoration: underline !default;
-
-
-//== Typography
-//
-//## Font, line-height, and color for body text, headings, and more.
-
-$font-family-serif:       "Times New Roman", Georgia, Times, serif !default;
-//** Default monospace fonts for `<code>`, `<kbd>`, and `<pre>`.
-$font-family-monospace:   Menlo, Monaco, Consolas, "Courier New", monospace !default;
-$font-family-base:        $font-family-sans-serif !default;
-
-$font-size-large:         ceil(($font-size-base * 1.25)) !default; // 23px
-$font-size-small:         ceil(($font-size-base * 0.85)) !default; // 16px
-$font-size-xsmall:        ceil(($font-size-base * 0.78)) !default; // 14px
-
-$font-size-h1:            floor(($font-size-base * 2.6)) !default; // 46px
-$font-size-h2:            floor(($font-size-base * 2.15)) !default; // 38px
-$font-size-h3:            ceil(($font-size-base * 1.7)) !default; // 31px
-$font-size-h4:            ceil(($font-size-base * 1.25)) !default; // 23px
-$font-size-h5:            $font-size-base !default;
-$font-size-h6:            ceil(($font-size-base * 0.85)) !default; // 16px
-
-//** Computed "line-height" (`font-size` * `line-height`) for use with `margin`, `padding`, etc.
-$line-height-computed:    floor(($font-size-base * $line-height-base)) !default; // ~30px
-
-//** By default, this inherits from the `<body>`.
-$headings-font-family:    $font-family-base !default;
-$headings-font-weight:    400 !default;
-$headings-line-height:    1.1 !default;
-$headings-color:          inherit !default;
+$font-family-sans-serif: "Noto Sans",'Noto Sans TC', "Apple LiGothic Medium", Roboto, "Microsoft JhengHei", "Microsoft JhengHei", "新細明體", "Lucida Grande", "Lucida Sans Unicode", sans-serif;
```

Comment:
I found maybe we should change it as:

"Noto Sans TC", "Noto Sans", "PingFang TC", "Apple LiGothic Medium", Roboto, "Microsoft JhengHei", "Lucida Grande", "Lucida Sans Unicode", sans-serif;